bigquery/datalab-migration/noxfile_config.py

```diff
@@ -23,7 +23,8 @@
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
     # Skipping for Python 3.9 due to pyarrow compilation failure.
```

Comment:
We shouldn't have to skip 3.9 anymore, as pyarrow wheels are out for Python 3.9 and 3.10 now.